### ISSUE TITLE
Make sure code recognizes a third reviewer if they exist. 

### ISSUE THREAD

**Issue by lwasser** [pyOpenSci/pyosMeta#77]
* what happens if a reviewer is NOT on github?? there will be no @ associated with the username 
* what happens if there is a reviewer 3:
* what happens if there is text after the @username in the issue (it fails)

**Comment by willingc** [comment]
From the `src/update_review_teams.py`:

```
# TODO - FEATURE we have some packages that were NOT approved but we had
# editors and reviewers.
# We need to acknowledge these people as well. maybe tag them with waiting on
# maintainer response??
# TODO: package-wide feature: create no update flag for entries
# TODO: make sure we can add a 3rd or 4th reviewer - crowsetta has this as
# will biocypher
```

Also:
```
3. FUTURE: finally it looks to see if we are missing review participants from
the review issues in the contributor file and updates that file.
```

**Comment by lwasser** [comment]
thank you @willingc shall we turn this into a list and then i can make individual issues as we tackle items here?? I actually think that FUTURE: item was completed. i create a new workflow that updates both the packages list AND the contributor list with any review information. so we can likely remove that part. i totally see why having smaller issues would make more sense VS comments in the text.

you are just seeing what i do when i'm working by myself (which i understand isn't ideal!)

**Comment by lwasser** [comment]
i also tried to implement this one

package-wide feature: create no update flag for entries

but i couldn't figure out how to do it as when i import the yaml file with ruemal_yaml, it removes comments. but sometimes a user's info  isn't on github 

example: people who only list their first names there or have no names there. 

but we want to override that field - such as a first and last name and store the name - in our yaml file (with their permission).

**Comment by lwasser** [comment]
this was addressed via #174 as we now support the reviewers field!! closing
